src/lib/git/repo.ts

```diff
@@ -0,0 +1,223 @@
+import Arweave from 'arweave'
+import Dexie from 'dexie'
+import git from 'isomorphic-git'
+import { v4 as uuidv4 } from 'uuid'
+import { InjectedArweaveSigner } from 'warp-contracts-plugin-signature'
+
+import { CONTRACT_TX_ID } from '@/helpers/constants'
+import getWarpContract from '@/helpers/getWrapContract'
+import { toArrayBuffer } from '@/helpers/toArrayBuffer'
+import { waitFor } from '@/helpers/waitFor'
+import { withAsync } from '@/helpers/withAsync'
+
+import { checkoutBranch, getCurrentBranch } from './branch'
+import { FSType } from './helpers/fsWithName'
+import { packGitRepo, unpackGitRepo } from './helpers/zipUtils'
+
+const arweave = new Arweave({
+  host: 'ar-io.net',
```

Comment:
Can we verify with the ario team that this gateway is more stable than arweave.net?

Reply:
Dropped a message asking their team about this.